namerd/core/src/main/scala/io/buoyant/namerd/Namerd.scala

```diff
@@ -1,8 +1,8 @@
 package io.buoyant.namerd
 
-import com.twitter.finagle.ListeningServer
+import com.twitter.finagle.{Path, Namer, ListeningServer}
 
-case class Namerd(interfaces: Seq[Servable], dtabStore: DtabStore)
+case class Namerd(interfaces: Seq[Servable], dtabStore: DtabStore, namers: Seq[(Path, Namer)])
```

Comment:
[S] namers could be a `Map[Path, Namer]` and you could move the `.toSeq` from NamerdConfig.scala into NamerdAdmin.scala